Libraries/Types/CoreEventTypes.js

```diff
@@ -88,7 +88,7 @@ export type TextLayoutEvent = SyntheticEvent<
 export type PressEvent = ResponderSyntheticEvent<
   $ReadOnly<{|
     changedTouches: $ReadOnlyArray<$PropertyType<PressEvent, 'nativeEvent'>>,
-    force: number,
+    force: ?number,
```

Comment:
Since this key might not be present the question mark should be on the left side. As you have it right now it says "force will always be there, but the value may be undefined, null, or a number." What you want it to say is "force may not be there, but if it is it will be a number". That is written as `force?: number`.

Reply:
So `target` is also wrong?

Reply:
I'm not sure. It's possible `target` is intending the behavior it says.

Reply:
I think that I had saw an event without `target` during earlier tests, but I have no repro right now. I will open another PR if I will be able to confirm this.